Sources/CreateProject/CreateProject.swift

```diff
@@ -142,6 +147,27 @@ private func createProjectPathDirectoryIfNeeded(at projectPath: String) throws {
     }
 }
 
+func format(targetName: String) -> String {
+    var newName = ""
+    let badCharacter: Set<Character> = [" ", "-"]
+    // Upper case first one
+    var uppercaseNext = true
+    
+    for character in targetName {
+        if badCharacter.contains(character) {
+            uppercaseNext = true
+            continue
+        }
+        if uppercaseNext {
+            newName.append(character.uppercased())
+            uppercaseNext = false
+            continue
+        }
+        newName.append(character)
+    }
+    return newName
+}
```

Comment:
Can you tell me more about this function?
Why is "-" a bad character?

I'm wondering if it's because of Godot, Swift Package Manager, or the library files, perhaps it's OS dependent.

Reply:
It's a swift limitation. Swift libraries cannot have a `-` or ` ` in the name. It will be an error when we try to import it.